packages/apm/src/integrations/express.ts

```diff
@@ -61,61 +65,65 @@ function wrap(fn: Function): RequestHandler | ErrorRequestHandler {
 
   switch (arrity) {
     case 2: {
-      return function(this: NodeJS.Global, _req: Request, res: Response): any {
-        // tslint:disable: no-unsafe-any
-        const transaction = (res as any).getTransaction && (res as any).getTransaction();
+      return function(this: NodeJS.Global, _req: Request, res: Response & SentryTracingResponse): any {
+        const transaction = res.__sentry_transaction;
         if (transaction) {
           const span = transaction.startChild({
             description: fn.name,
             op: 'middleware',
           });
           res.once('finish', () => {
-            span.finish();
+            if (span) {
```

Comment:
Unnecessary `if`. When we are in `if (transaction)` block, it'll always be true.